docs-rtd/howto/manage-migrations.md

```diff
@@ -0,0 +1,101 @@
+(manage-migrations)=
+
+# Manage migrations
+
+The following document shows you what changes, if any, are needed to your Terraform plan when migrating a model(s) between Juju controllers and/or JAAS.
+
+## Migrating to another Juju controller
+
+When a model is migrated to a new Juju controller, no changes are needed. Simply update you plan to connect to the new controller.
+
+The only exception to this scenario is when cross-model relations are involved. If both models involved in a cross-model relation are moved, no changes are necessary. If only one model involved in the relation is moved, see below.
+
+(manage-migration-cross-controller-relations)=
+
+### Cross-controller relations
+
+When a model consumes an offer from another model, this is known as a cross-model relation.
+If the providing model is moved, your applications will continue to work but the Juju provider does not currently support the creation of cross-controller relations.
+
```

Comment:
This is unexpected because in Juju a CMR is cross-model. Same controller or not doesn't really matter. We should record this as a limitation of CMR in TFJuju, that is, in here: https://documentation.ubuntu.com/terraform-provider-juju/latest/howto/manage-relations/#add-a-cross-model-relation and https://documentation.ubuntu.com/terraform-provider-juju/latest/howto/manage-offers/#integrate-with-an-offer  The Migrate a model section should be able to just point there.